backend/python/app/__init__.py

```diff
@@ -1,12 +1,35 @@
 import os
+import firebase_admin
 
 from flask import Flask
 from flask_sqlalchemy import SQLAlchemy
+from logging.config import dictConfig
 
 from .config import app_config
 
 
 def create_app(config_name):
+    # configure Flask logger
+    dictConfig(
+        {
+            "version": 1,
+            "handlers": {
+                "wsgi": {
+                    "class": "logging.FileHandler",
+                    "level": "ERROR",
+                    "filename": "error.log",
+                    "formatter": "default",
+                }
+            },
+            "formatters": {
+                "default": {
+                    "format": "%(asctime)s-%(levelname)s-%(name)s::%(module)s,%(lineno)s: %(message)s"
+                },
+            },
+            "root": {"level": "ERROR", "handlers": ["wsgi"]},
+        }
+    )
```

Comment:
This configures Flask's built-in logger for the entire application, please pass the application logger instance into your services @alexguo8 @chenrichard10 

e.g.
```python

# controller
from flask import current_app

user_service = UserService(current_app.logger)


# service
class UserService:
    def __init__(self, logger):
        self.logger = logger

    def func(self):
        self.logger.error("Error occurred...")
```